app/views/patients/show.html.erb

```diff
@@ -126,10 +130,14 @@
           <td><%= cc.last_name %></td>
           <td><%= cc.primary_telephone.present? && cc.primary_telephone.length > 10 ? cc.primary_telephone&.gsub('+1', '')&.gsub(/\D/, '')&.insert(3, '-')&.insert(7, '-') : '' %></td>
           <td><%= cc.email %></td>
-          <td><%= cc.contact_attempts %></td>
+          <td class="text-center"><%= cc.last_date_of_exposure&.to_date&.strftime('%m/%d/%Y') || 'Unknown' %></td>
+          <td class="text-center"><%= cc.assigned_user %></td>
```

Comment:
Why did we add "text-center" to these columns? It makes it inconsistent with the the other columns already here.

Reply:
Because i thought it looked really bad without them. But im happy to revert it to make it consistent.
<img width="265" alt="image" src="https://user-images.githubusercontent.com/17532163/106168748-a353dd80-615c-11eb-9f06-82a074500891.png">

Reply:
Yes please revert for consistency for now.

Reply:
Will do.